lib/core.sh

```diff
@@ -142,6 +142,7 @@ function _get_remote_command(){
     local rc_script="$(_concatenate_files "$KYRAT_HOME"/bashrc "$KYRAT_HOME"/bashrc.d/* | $GZIP | $BASE64)"
     local inputrc_script="$(_concatenate_files "$KYRAT_HOME"/inputrc "$KYRAT_HOME"/inputrc.d/* | $GZIP | $BASE64)"
     local vimrc_script="$(_concatenate_files "$KYRAT_HOME"/vimrc "$KYRAT_HOME"/vimrc.d/* | $GZIP | $BASE64)"
+    local tmux_conf="$(_concatenate_files "$KYRAT_HOME"/tmux.conf "$KYRAT_HOME"/tmux.conf.d/* | $GZIP | $BASE64)"
```

Comment:
In Line 63 will still be helpful to init the directory `tmux.conf.d`, similarly to what we do for `bashrc`, `inputrc` and `vimrc`.

Reply:
Thanks, I overlooked this.